exec/java-exec/src/main/java/org/apache/drill/exec/rpc/user/UserServer.java

```diff
@@ -255,11 +257,12 @@ void disableReadTimeout() {
       getChannel().pipeline().remove(BasicServer.TIMEOUT_HANDLER);
     }
 
-    void setUser(final UserToBitHandshake inbound) throws IOException {
+    void setUser(final UserToBitHandshake inbound, Map<String, String> sessionParams) throws IOException {
```

Comment:
Can you give example of sessionParameters? How they are different from UserProperties? Can we combine them?